config/config.php

```diff
@@ -14,4 +14,7 @@
     'output_transform' => [
         'App\\Enums\\' => '',
     ],
+
+    // Whether the output files should end with .ts.
+    'ts_output_files' => false,
```

Comment:
How about if this is instead ```output_file_extension``` and by default it's ```js```? That was if there's any other usecase for a different file extension, it'll be supported.

Reply:
Sounds logical. Should it contain the dot or no?

This?
```javascript
    // Set a specific extension for the output files (prepended with a dot).
    'output_file_extension' => '.js',
```

Or this?
```javascript
    // Set a specific extension for the output files (without a dot character).
    'output_file_extension' => 'js',
```

Reply:
Let's go without the dot, just ```js```

Reply:
Ok! Changed.